script.js

```diff
@@ -0,0 +1,91 @@
+const loginForm = document.querySelector('#login-form');
+const loginButton = document.querySelector('#login-btn');
+
+loginButton.addEventListener('click', event => {
+  event.preventDefault();
+  handleLogin();
+});
+
+function handleLogin() {
+    // get the username entered by the user
+    const usernameInput = document.querySelector('#username');
+    const username = usernameInput.value;
+  
+    // connect to Scaledrone using the user's username as the client ID
+    const drone = new ScaleDrone('gM860Lw7cfNCaIcl', {
```

Comment:
ok, i got that part, teacher told us to put chanell ID  in .env file because it must be hidden :😂

Reply:
.env works for backend stuff. Since this is client code that gets downloaded to a browser a malicious hacker can always find this. There's no way to hide secrets in client code, apart from trying to obfuscate it as much as possible.